### DISCUSSION TITLE
In desperate need of advice

### DISCUSSION THREAD

**Discussion by lissmiss2018** [community/community#21910] [New to GitHub]
<p>This is SUCH a long story and I’ve wanted to come and ask for help months ago after Apple wasn’t…but I’m not sure if this is the correct place for me to ask or if I should at all I’ll just be quick (and sorry to bug you all)…</p>
<p>I’ve been getting hacked and harrassed daily since September and I can’t take much more of it. Any advice or help would be appreciated…not sure how I’m to try and explain it all over the internet but I’ve tried everything. Thanks for listening and hope you all have a great day!</p>
<p>Liss</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>I’m sorry that you’ve been dealing with that. And don’t worry, you’re not being a bother. Unfortunately, I’m not sure that we can help specifically. If you’re having problems specifically on GitHub then please contact our support teams directly and privately via the <a href="https://github.com/contact" rel="nofollow noopener">contact form</a> or email.</p>
<p>On the other hand, it sounds like what you’re experiencing is more pervasive than that. If so, you should contact the <a href="http://www.crashoverridenetwork.com/" rel="nofollow noopener">Crash Override Network</a>. They specifically deal with online abuse and are probably going to be your best resource for a comprehensive harassment campaign.</p>
<p>Please let us know if we can help further.</p>

**Comment by lissmiss2018** [comment]
<p>Oh thanks so much for the quick reply!  No, it’s not anyone from here…I honestly don’t know who it is but I have piles of snippets of the abuse, threats, and HDD full of my business Mac that I saw some scary stuff that I copied right before I was booted for good (all with all my years of work).  I’ve gone through 4 computers, 3 iPhones, 2 android and 2 iPads and it’s beating me down.  I can’t do anything online and I have found a new passion after researching but anything I do, I’m watched, shut/kicked out, etc,etc.  (wow…I sound nuts! Lol)</p>
<p>Sadly that’s not the worst of it and I have so much evidence right when it started that my devices were being hacked remotely but Apple, my carrier, government, all the people who I assumed would help just said “not possible” which only made things worse.  But there’s so much information to learn that I new I wasn’t sophisticated enough in the coding world to stand a chance and, seriously today all my fight left.  But I appreciate your suggestion because it’s an avenue I have tried, or even new of!  I’m incredibly grateful for the quick response and for you’re advice.  Feel a little less deflated now!  Thanks so much!  </p>

**Comment by lee-dohm** [comment]
<p>You’re most welcome. I wish you luck in getting this resolved for good!</p>
